static/tests/reducers/auth.test.js

```diff
@@ -0,0 +1,112 @@
+/* global describe, test, expect */
+import createReducer from '../../src/reducers/auth';
+import {
+    LOGIN_USER_SUCCESS,
+    LOGIN_USER_FAILURE,
+    LOGIN_USER_REQUEST,
+    LOGOUT_USER,
+    REGISTER_USER_FAILURE,
+    REGISTER_USER_REQUEST,
+    REGISTER_USER_SUCCESS,
+} from '../../src/constants/index';
+
+describe('Auth reducer', () => {
+  const initialState = {
+    token: null,
+    userName: null,
+    isAuthenticated: false,
+    isAuthenticating: false,
+    statusText: null,
+    isRegistering: false,
+    isRegistered: false,
+    registerStatusText: null,
+  };
+  it('should return the initial state', () => {
+      const returnedData = createReducer(undefined, {});
```

Comment:
Can give this one initial state too. It's testing that initialising a reducer with initial state returns the initial state and doesn't mutate data